NuKeeper.Git/LibGit2SharpDriver.cs

```diff
@@ -95,7 +97,11 @@ public Task Checkout(string branchName)
                 {
                     if (BranchExists(branchName))
                     {
-                        GitCommands.Checkout(repo, repo.Branches[branchName]);
+                        _logger.Normal($"Git checkout local branch '{branchName}'");
+                        // Some files are automatically generated in /obj folder.
+                        // If there is no .gitignore there will be conflicts because of this and you cannot change branches
+                        // CheckoutModifiers.Force makes that all changes are ignored.
+                        GitCommands.Checkout(repo, repo.Branches[branchName], new CheckoutOptions { CheckoutModifiers = CheckoutModifiers.Force });
```

Comment:
> This will throw away local changes.

This seems bad you could delete someones work if they haven't stashed it, I would rather throw an exception at this point possibly advising a .gitignore

Reply:
Will check if I can check for that specific exception